src/main/java/org/jvnet/hudson/test/JenkinsRule.java

```diff
@@ -252,6 +183,79 @@
 import org.springframework.dao.DataAccessException;
 import org.xml.sax.SAXException;
 
+import javax.servlet.ServletContext;
```

Comment:
Prefer strictly alphabetized `import`s in one block. Not for humans to read anyway.

Reply:
Ok, I have updated the files changed by this PR.